shop/src/test/java/ro/msg/learning/shop/OrderServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package ro.msg.learning.shop;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import ro.msg.learning.shop.configuration.OrderServiceTestConfiguration;
+import ro.msg.learning.shop.exception.entity_exception.LocationException;
+import ro.msg.learning.shop.model.*;
+import ro.msg.learning.shop.repository.*;
+import ro.msg.learning.shop.service.OrderService;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@DataJpaTest
+@Import(OrderServiceTestConfiguration.class)
+class OrderServiceTest {
```

Comment:
What kind of test is this one? If it is integration test then you have to call the endpoints using mockMvc.perform, not the service or repository methods. 

Also, for test data it was suggested in the training material to have a Test Controller activated just for testing purposes and in which you have a populate method (to populate the database with test data needed)

Reply:
no, the integration tests are coming. it wasn't this one